### ISSUE TITLE
Running multiple worker instances

### ISSUE THREAD

**Issue by srolija** [judge0/api#72]
Hi,
I am interested in how to run multiple workers s on multiple machines. Looking at the compose, it should be fairly easy to run workers one some machines and API + db + redis on other. 

I figured out that I would need to change the `REDIS_HOST` and database credentials in `judge0-api.conf` for each worker. Looking at the app, it seems it is only communicating with workers via the `resque` tasks. If that is the case, and given how compose for workers just runs `resque:work`, it seems it should be it. 

So in a nutshell, I am not missing something I should just:
1. Start API (with redis and database)
2 . Run workers on separate machines with the same config files pointing to the started Redis instance.

Thanks for this great project! :)

**Comment by hermanzdosilovic** [comment]
Hi @srolija,

Sorry for the delayed answer.

Here is one way you could do this.

On one machine run API, Postgres and Redis. It is important that Postgres and Redis expose their ports. Example Compose file and default configuration for this is [here](https://github.com/judge0/api/files/3298983/judge0-api-v1.2.0.zip)

On other N machines you run just worker that through variables `REDIS_HOST` and `POSTGRES_HOST` knows where his Postgres and Redis are. Example Compose file and configuration that has just these variables changed is [here](https://github.com/judge0/api/files/3298986/judge0-api-v1.2.0-worker.zip).

Let me know if this works for you and if you have any more questions about this setup.
